src/exberry_int/integration_exberry.py

```diff
@@ -361,22 +373,33 @@ def mass_cancel(mass_cancel_req):
         return mass_cancel_json
 
 
-
     async def connect():
-        LOG.info(f"Connecting to the Exberry Trading API at {env.tradingApiUrl} ...")
-        ws = await ClientSession().ws_connect(env.tradingApiUrl)
-        LOG.info("...Connected to the Exberry Trading API")
+        ws = None
+        tasks = []
+        try:
+            LOG.info(f"Connecting to the Exberry Trading API at {env.tradingApiUrl} ...")
+            ws = await ClientSession().ws_connect(env.tradingApiUrl)
+            LOG.info("...Connected to the Exberry Trading API")
```

Comment:
Does exberry send any kind of initial message when a new connection is open? You might want to log that too...

Reply:
Currently, they do not in the form of a message